releasenotes/notes/beta-8-catchup-594571ea1a9ff021.yaml

```diff
@@ -0,0 +1,19 @@
+---
+features:
+  - Logs-agent now runs as a goroutine in the main agent process
+  - All docker event subscribers are multiplexed in one connection, reduces stress on the docker daemon
+  - The Agent can find relevant listeners on its host by using the "auto" listener, for now only docker is supported
+security:
+  - "[windows] In MSI installer, sign only datadog binaries"
```

Comment:
Is it useful in the changelog ?

Reply:
synced with @derekwbrown, and according to him that's much more an internal change than customer-facing, so I'll go ahead and remove this